src/app/learn/reference/configuration/additional-config-files/index.html

```diff
@@ -83,11 +83,15 @@
   </ul>
 
   <stache-page-anchor>
-    Default <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> for local development
+    <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> settings
   </stache-page-anchor>
 
   <p>
-    If you need to track the <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> querystring parameters and expose them through the <stache-code>SkyAppConfig</stache-code> for local development, we recommend creating a <stache-code>skyuxconfig.serve.json</stache-code> file and adding the <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> settings to the <stache-code>params</stache-code> property. In addition, if you use the global nav exclusively and don’t want the local nav alongside it, you can set the <stache-code>routes</stache-code> property to <stache-code>false</stache-code>.
+    To track <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> querystring parameters and expose them through <stache-code>SkyAppConfig</stache-code> for a particular CLI command, create a <stache-code>skyuxconfig.[COMMAND].json</stache-code> file and add the <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> settings to the <stache-code>params</stache-code> property.
```

Comment:
Let's schedule a few minutes to quickly talk through this one John.  

It's kind of tricky since we use the technique to both tell builder to read the parameters from the querystring as we do to set the default value.  The other interesting bit we should probably call out is that `addin`, `envid`, `svcid`, and `leid` are automatically configured to be read through the QS.

https://github.com/blackbaud/skyux-builder/blob/master/skyuxconfig.json#L10-L14